hw02_unpack_string/go.mod

```
@@ -1,4 +1,4 @@
-module github.com/fixme_my_friend/hw02_unpack_string
+module github.com/Drodik/otus-go/tree/master/hw02_unpack_string
```

```suggestion
module github.com/Drodik/otus-go/hw02_unpack_string
```

`/tree/master/` - не относится к пути модуля, это UI GitHub